resources/js/pages/users/create.tsx

```
@@ -1,29 +1,29 @@
 import { useLanguage } from '@/hooks/use-language';
 import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
 import { Button } from '@/components/ui/button';
 import { Input } from '@/components/ui/input';
 import { Label } from '@/components/ui/label';
-import { router } from '@inertiajs/react';
 import { useForm } from '@inertiajs/react';
 import { toast } from 'react-hot-toast';
 import { cn } from '@/lib/utils';
-import { RolePermission } from '@/components/role-permission';
+import { UserRoleSelect } from '@/components/user-role-select';
+import { RolePermissionManager } from '@/pages/users/partials/role-permission-manager';
```

ESLint

[error] 'RolePermissionManager' is defined but never used.

(@typescript-eslint/no-unused-vars)